.clang-tidy

```diff
@@ -14,6 +14,7 @@ Checks: >
   performance-*,
   -performance-avoid-endl,
   readability-*,
+  -readability-magic-numbers,
```

Comment:
What magic numbers are being complained about?

Reply:
The 2.0 [here](https://github.com/codeplaysoftware/portFFT/blob/4976174f0dd06c3fe162b7b715257e81d6b3827e/src/portfft/committed_descriptor_impl.hpp#L433)
I do not want to especially name it.

Reply:
I think you should disable the check around the 1 line ([stackoverflow](https://stackoverflow.com/questions/37950439/inline-way-to-disable-clang-tidy-checks)).

This might alternatively be fixed by using the log2 with ints as suggested in another comment.